lib/libc/include/libc_private.h

```diff
@@ -395,7 +395,8 @@ int __strerror_rl(int errnum, char *strerrbuf, size_t buflen,
 	    struct _xlocale *locale);
 
 #ifdef CHERI_LIB_C18N
-__pid_t		__sys_rfork(int);
+int	_rtld_c18n_is_enabled(void);
```

Comment:
Should this return a bool?

Reply:
bool doesn't compile when libc_private.h is included in lots of files in libc.

Reply:
You could just include <stdbool.h> in libc_private.h if you wanted that to work.